### ISSUE TITLE
Assemble inc/dec of address

### ISSUE THREAD

**Issue by skx** [skx/assembler#16]
We need to assemble:

     inc byte ptr [rax]

And:

     dec byte ptr [rax]

This is required for the [brainfuck compiler](https://github.com/skx/bfcc/), although we actually emit `add byte ptr [r8],NNN` we can just replace that with a series of inc/dec so implementing this will suffice.

Assembling is trivial, the harder part will be parsing:

```
0:  fe 00                   inc    BYTE PTR [rax]
2:  fe 03                   inc    BYTE PTR [rbx]
4:  fe 01                   inc    BYTE PTR [rcx]
6:  fe 02                   inc    BYTE PTR [rdx]
```

i.e. `fe` + register-number from `getreg`.  `dec` is the same, but with the 0x8 + N:

```
0:  fe 08                   dec    BYTE PTR [rax]
2:  fe 0b                   dec    BYTE PTR [rbx]
4:  fe 09                   dec    BYTE PTR [rcx]
6:  fe 0a                   dec    BYTE PTR [rdx]
```

**Comment by skx** [comment]
See here for different size/types:

```
0:  fe 08                   dec    BYTE PTR [rax]
2:  66 ff 08                dec    WORD PTR [rax]
5:  48 ff 08                dec    QWORD PTR [rax]
```

Start with the lexer/parser.  We need `[` `]` and `byte`,`word`, `qword`, and `ptr` as tokens.

**Comment by skx** [comment]
Closed by #17.
